src/System.Private.CoreLib/shared/System/Runtime/Intrinsics/Vector128_1.cs

```diff
@@ -201,6 +233,23 @@ public Vector128<T> WithElement(int index, T value)
             return result;
         }
 
+        /// <summary>Gets the hash code for the instance.</summary>
+        /// <returns>The hash code for the instance.</returns>
+        /// <exception cref="NotSupportedException">The type of the current instance (<typeparamref name="T" />) is not supported.</exception>
+        public override int GetHashCode()
+        {
+            ThrowIfUnsupportedType();
+
+            int hashCode = 0;
+
+            for (int i = 0; i < Count; i++)
+            {
+                hashCode = HashHelpers.Combine(hashCode, GetElement(i).GetHashCode());
```

Comment:
Is `HashHelpers` still the recommended thing to use internally?

Reply:
I'm not sure, but we have a public API for combining hash codes now:

[https://github.com/dotnet/coreclr/blob/master/src/System.Private.CoreLib/shared/System/HashCode.cs](https://github.com/dotnet/coreclr/blob/master/src/System.Private.CoreLib/shared/System/HashCode.cs)

---
In reply to: [237296343](https://github.com/dotnet/coreclr/pull/21259#discussion_r237296343) [](ancestors = 237296343)

Reply:
We also have issues like this:  [https://github.com/dotnet/coreclr/issues/15508](https://github.com/dotnet/coreclr/issues/15508) "Replace HashHelpers.Combine with HashCode.Combine"

---
In reply to: [237297239](https://github.com/dotnet/coreclr/pull/21259#discussion_r237297239) [](ancestors = 237297239,237296343)

Reply:
Fixed.